documentation/docs/faq.md

```diff
@@ -10,4 +10,27 @@ keywords:
 - IOTA Identity
 ---
 
-TODO: Add list of frequently asked questions and there answers
\ No newline at end of file
+This page contains frequently asked questions regarding the Identity Library and Self Sovereign Identity in general.
+
+### What programming languages are supported by the identity library?
+Please see the "Programming Languages" section, to find out which languages are available.
+### Do I need to have IOTA tokens to start building with identity?
+At the moment you don't need IOTA tokens to create and manage identities, although we are looking at opportunities to utilize the token in the future. 
+### How do I prove control over my DID?
+Control over an identity is ultimately tied to the control over cryptographic key material (something you have).
```

Comment:
Maybe we should mention the DIDComm Authentication protocol here? Or should it be vague since that is not implemented yet?

Reply:
I think keeping it vague for now is the way to go, otherwise it would raise to many expectations.